src/main/scala/com/mayreh/thankyou/SbtPlugin.scala

```diff
@@ -19,7 +19,7 @@ object SbtPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     thankYouStars := {
-      val updateReport = update.value
+      val updateReport = updateFull.value
```

Comment:
Hi, could you elaborate a bit why this change is necessary?
Also I suppose this doesn't work on sbt 0.13 so we should drop 0.13 support from now on right?

Reply:
The `update` task is used to collect `.callers` that was empty. I searched for it and found https://github.com/sbt/sbt/issues/4706#issuecomment-493680547 which led me to try `updateFull` instead and it worked. 🤷🏼‍♂️

Reply:
I see, got it.

Would you mind also update here to include only `1.6.2` ? (https://github.com/ocadaruma/sbt-thank-you-stars/blob/master/build.sbt#L11)

Reply:
done